crates/bevy_ecs/src/system/commands.rs

```diff
@@ -158,12 +158,25 @@ impl<'a> Commands<'a> {
 
     /// Equivalent to iterating `bundles_iter` and calling [`Self::spawn`] on each bundle, but
     /// slightly more performant.
-    pub fn spawn_batch<I>(&mut self, bundles_iter: I)
+    pub fn spawn_batch<I>(&mut self, bundles_iter: I) -> impl IntoIterator<Item = Entity>
```

Comment:
This function returns a `Vec`. Exposing it directly to the user would allow avoiding `.into_iter().collect()` in case the user wanted to store it in a `Vec`.